snap/hooks/configure

```diff
@@ -643,6 +643,7 @@ fi
 if [ -e "${SNAP_DATA}/var/lock/no-flanneld" ]
 then
   touch "${SNAP_DATA}/var/lock/cni-needs-reload"
+  touch "${SNAP_DATA}/var/lock/try-offloading-patch"
```

Comment:
patch offloading is a one-time configuration, so i'd suggest moving this to `install` instead.